spaces/qubcoins/skin.scss

```
@@ -0,0 +1,9 @@
+.sushi {
```

The css selector should similar to the space key.
```suggestion
.qubcoins {
```